sample/db/samples/variants.rb

```diff
@@ -30,184 +30,218 @@
     product: solidus_tshirt,
     option_values: [small, blue],
     sku: "SOL-00003",
-    cost_price: 17
+    cost_price: 17,
+    price: 10
```

Comment:
it seems a bit strange that `price` is lower than the `cost_price`, shouldn't it be greater than that?

Reply:
Fair enough, fixed :sweat_smile: